src/psalter.php

```diff
@@ -117,6 +118,9 @@ function ($arg) use ($valid_long_options, $valid_short_options) {
 
     --codeowner=[codeowner]
         You can specify a GitHub code ownership group, and only that owner's code will be updated.
+
+    --allow-backwards-incompatible-changes=BOOL
```

Comment:
Add a not to say it defaults to `true`

Reply:
Rather than a `--allow-backwards-incompatible-changes=<BOOL>` option who'ss omission is equivalent to  `--allow-backwards-incompatible-changes=true`, I think it might be more intuitive to have a  `--no-backwards-incompatible-changes` option that doesn't take a value.